.gitignore

```diff
@@ -36,4 +36,4 @@ certs/
 /private
 
 # Ignore config which may contain secrets
-**/openid_connect.client.azure_b2c.yml
```

Comment:
I think we should keep all openid_connect secrets in ignore. Maybe expand this to `**/openid_connect.client.*.yml`? That way, people still can't do an export and accidentally commit